### DISCUSSION TITLE
Election of a new leader of the MySQL working group

### DISCUSSION THREAD

**Discussion by bmalynovytch** [ansible-collections/ansible.mysql#559] [General] [answered]
https://github.com/ansible-collections/community.mysql/wiki/MySQL-Working-Group

Hi folks !

I've been messing around and contributing for Ansible for years (started when Ansible was still in version 1.8).
As an active contributor of the MySQL plugins in the early ages, I became the lead of this working group, with the great help of @Andersson007.

I'm also an active user of RHEL derived distributions, using CentOS intensively for years until the distro switched to Stream which I disagreed. I then switched to Rocky Linux like most of the community disappointed by RedHat's decision.

Recently, RedHat decided to close RHEL's code completely, which means that the community won't be able to use the distributions for which it contributes anymore.

That's why I decided not to contribute anymore in any way to RedHat's products, including Ansible.

I therefore return the symbolic crown 👑 of the Lead of this working group and wish you the best for the future.
I really enjoyed working with you guys, but it's time for me to write another chapter of my OSS contribs.

@Andersson007  @laurent-indermuehle @Jorge-Rodriguez @markuman @rsicart

**Comment by Andersson007** [comment]
@bmalynovytch hello and, oh, sad to hear that

I won't try to persuade you that this collection since the split of ansible/ansible is a fully community undertaking not supported by RedHat (I'm participating in my spare time too) but, yeah, feeling like I'm persuading.. :)

Anyway, thanks a lot for all those super valuable contributions (code, review, advice and other kinds) you did along the years.
It was a pleasure to work with you and talk to you..

If once you change your mind, we'll be happy to see you back as a leader or contributor if you don't want to be the leader.

Thanks and good luck!

**Comment by Andersson007** [comment]
FYI: I left the row in [wiki](https://github.com/ansible-collections/community.mysql/wiki/MySQL-Working-Group) blank. Will keep the discussion open for a while in case anyone is interested.
1. Folks if someone is interested to become a leader, please speak up.
2. We could decide on some benefits for our leader like, say, a tie-breaking vote or whatever.
3. Any other suggestions, like not having a leader whatsoever

**Comment by bmalynovytch** [comment]
Even though MySQL isn't what you know the best @Andersson007, I think you should be the new "leader" / tie-breaker, whatever you'll all name it. Your investment in the project has always be highly valuable and you always try to make things move forward the good way, taking advises from others.

**Comment by eRadical** [comment]
Sorry to see you go @bmalynovytch! Thank you for your hard work. Good luck in your future work.

I also incline towards @Andersson007.
Up until now I was just "watching" this repository as I don't have that much experience w/ Python (but, slowly, getting there).
I have enough MySQL/MariaDB knowledge and I'll make some time to help more!

**Comment by laurent-indermuehle** [comment]
Sorry @bmalynovytch and thank you for your legacy in this repo :)

If I had to choose a leader, I would name @Andersson007 without any hesitation. For his wisdom, kindness and presence!

**Comment by rsicart** [comment]
Hi folks,

Sorry to read that, but I understand. Thanks for your contributions @bmalynovytch and good luck with your new projects.

Otherwise I think that @Andersson007 should be the new team leader.

See you around!

**Comment by Andersson007** [comment, marked as answer]
Ah, friends, I'm really touched with what you all are saying!
I'll put myself in that sheet then (not because I want but because of your trust).
If anyone later wants to be a leader, please open a discussion.
It's a pleasure to work with you all and it's great to see our community growing! Thank you!
UPDATE: and sad to see people like Benjamin leaving..

**Comment by gundalow** [comment]
@bmalynovytch Thank you for the amazing work you've done here. I hope our paths will cross again in the future
